iodata/formats/mol2.py

```diff
@@ -0,0 +1,152 @@
+from ..utils import angstrom, LineIterator
+
+
+__all__ = []
+
+
+PATTERNS = ['*.mol2']
+
+
+@document_load_one("MOL2", ['atcoords', 'atnums', 'atcharges', 'atffparams'], ['title'])
+def load_one(lit: LineIterator) -> dict:
+    """Do not edit this docstring. It will be overwritten."""
+    molecule_found = False
+    while True:
+        try:
+            line = next(lit)
+        except StopIteration:
+            break
+        if len(line) > 1:
+            words = line.split()
+            if words[0] == "@<TRIPOS>MOLECULE":
+                # Found another molecule; go one line back and break
+                if molecule_found:
+                    lit.back(line)
+                    break
+                title = next(lit).strip()
+            if words[0] == "@<TRIPOS>ATOM":
+                atnums, atcoords, atchgs, attypes = _load_helper_atoms(lit)
+                atcharges = {"mol2": atchgs}
+                atffparams = {"attypes": attypes}
+                result = {
+                    'atcoords': atcoords,
+                    'atnums': atnums,
+                    'atcharges': atcharges,
+                    'atffparams': atffparams
+                }
+                if title is not None:
+                    result['title'] = title
+                molecule_found = True
+    if molecule_found is False:
+        raise lit.error("Molecule could not be read")
+    return result
+
+
+def _load_helper_atoms(lit: LineIterator) -> Tuple[np.ndarray, np.ndarray, np.ndarray, tuple]:
+    """Load element numbers, coordinates and atomic charges."""
+    atnums = []
+    atcoords = []
+    atchgs = []
+    attypes = []
+    for line in lit:
+        words = line.split()
+        if len(words) <= 6:
+            break
+        # Assume that the first character of atom type is the element
+        try:
+            atnums.append(sym2num.get(words[5][0].title()))
+        except ValueError:
+            print(f'Can not convert atom type {words[5][0]} to element')
+        attypes.append(words[5])
+        atcoords.append([float(words[2]), float(words[3]), float(words[4])])
+        if len(words) == 9:
+            atchgs.append(float(words[8]))
+        else:
+            atchg.append(0.0)
+    atnums = np.array(atnums, int)
+    atcoords = np.array(atcoords) * angstrom
+    atchgs = np.array(atchgs)
+    attypes = tuple(attypes)
+    return atnums, atcoords, atchgs, attypes
+
+
+@document_load_many("MOL2", ['atcoords', 'atnums', 'atcharges', 'atffparams'], ['title'])
+def load_many(lit: LineIterator) -> Iterator[dict]:
+    """Do not edit this docstring. It will be overwritten."""
+    # MOL2 files with more molecules are a simple concatenation of individual MOL2 files,'
+    # making it trivial to load many frames.
+    while True:
+        try:
+            yield load_one(lit)
+        except IOError:
+            return
+
+
+@document_dump_one("MOL2", ['atcoords', 'atnums'], ['atcharges', 'atffparams',  'title'])
```

Comment:
Pycodestyle complains because of the double space...
```suggestion
@document_dump_one("MOL2", ['atcoords', 'atnums'], ['atcharges', 'atffparams', 'title'])
```

Reply:
corrected